docs/new-architecture-app-renderer-ios.md

```diff
@@ -0,0 +1,128 @@
+---
+id: new-architecture-app-renderer-ios
+title: Enabling Fabric on iOS
+---
+
+:::caution
+
+This documentation is still **experimental** and details are subject to changes as we iterate. Feel free to share your feedback on the [react-native-website PR](https://github.com/facebook/react-native-website) for this page.
+
+Moreover, it contains several **manual steps**. Please note that this won't be representative of the final developer experience once the New Architecture is stable. We're working on tools, templates and libraries to help you get started fast on the New Architecture, without having to go through the whole setup.
+
+:::
+
+This section will go over how to enable the new renderer in your app. Make sure your application meets all the [prerequisites](new-architecture-app-intro).
+
+## 1. Enable Fabric in Podfile
+
+Add changes to your Podfile. You can see some examples in [RNTester](https://github.com/facebook/react-native/blob/main/packages/rn-tester/Podfile) and [rn-demo-app](https://github.com/facebook/fbt/blob/rn-demo-app/ios/Podfile).
+
+```ruby title="Podfile"
+# Add the following line at the top of Podfile.
+# Codegen produces files/classes that share names, and it will show the warning.
+# deterministic_uuids option surpresses the warning.
+install! 'cocoapods', :deterministic_uuids => false
+
+target 'Some App' do
+  pods()
+end
+
+def pods()
+# Get config
+config = use_native_modules!
+
+# Use env variables to turn it on/off.
+fabric_enabled = ENV['USE_FABRIC']
+use_codegen_discovery= ENV['USE_CODEGEN_DISCOVERY']
+
+# Enabled codegen discovery. This will run the codegen at preinstall time.
+# Files are generated at {pod installation root}/build/generated/ios/
+if use_codegen_discovery
+  Pod::UI.puts "[Codegen] Building target with codegen library discovery enabled."
+  pre_install do |installer|
+    use_react_native_codegen_discovery!({
+        react_native_path: config[:reactNativePath],
+        # Modify here if your app root path isn't the same as this one.
+        app_path: "#{Dir.pwd}/..",
+        fabric_enabled: fabric_enabled,
+    })
+  end
+end
+
+# Pass the flag to enable fabric to use_react_native!.
+use_react_native!(
+  ...
+  :fabric_enabled => fabric_enabled
+)
+```
+
+```bash
+$ ~/yourproject/**/ios; pod install
+```
```

Comment:
```suggestion
```

I'm not sure if running `pod install` here is required, since the guide recommends this again as a last step (4.).

If we want to leave this in, it might be worth adding an informative sentence above the command, for example: 
* "Then, update your project pods by running:"

Reply:
Yup the last step should be sufficient. Moreover here there is a missing `cd` so it's sort of confusing for the reader.